app/models/apple/podcast_container.rb

```diff
@@ -186,5 +195,27 @@ def podcast_deliveries_url
     def podcast_container_id
       id
     end
+
+    def files
+      apple_attributes.dig("files")
+    end
+
+    def has_podcast_audio?
+      return false if files.blank?
+
+      files.any? do |file|
+        # Retrieve the file status from the podcast container's files attribute
+        file["status"] == FILE_STATUS_SUCCESS && file["assetRole"] == FILE_ASSET_ROLE_PODCAST_AUDIO
+      end
+    end
+
+    def missing_podcast_audio?
+      !has_podcast_audio?
+    end
+
+    def needs_delivery?
+      # TODO: Overwriting the podcast audio with another file
+      missing_podcast_audio? && podcast_deliveries.empty?
```

Comment:
Okay, so here's the gatekeeper - it only needs delivery if the audio is missing _and_ we haven't tried deliveries yet.